### ISSUE TITLE
add ... to write.csv()

### ISSUE THREAD

**Issue by sachserf** [d-notebook/sheetr#1]
I would suggest to add `...`, to enhance options for write.csv (e.g. the string to use for missing values in the data, etc.)

**Comment by koundy** [comment]
Exactly.. I completely agree and wanted to suggest this too. For example, I wanted to use `row.names = F` option
